### PR TITLE
Feat actions

### PR DESCRIPTION
**Pull request by RobertGlein**
Add TE to CI

### CODE REVIEW

File: .github/workflows/hls.yml [hunk 1]
```diff
@@ -55,9 +55,15 @@ jobs: # A workflow run is made up of one or more jobs that can run sequentially
...
-# Does not work right now
-#    - name: TE # Script name
-#      run: |
-#        cd project
-#        source $VIVADO_PATH_SH
-#        vivado_hls -f "script_TE.tcl"
+    - name: TE # Script name
+      run: |
+        cd emData/TE
+        ./download.sh
```
Comment (pwittich) [inline .github/workflows/hls.yml]: do we need this? I think the goal is to have the repo and CI use the few data files that are already in the repo.

**Comment by pwittich** [comment]
also it looks like the MC test fails at line 500 but the overall test passed?

File: .github/workflows/hls.yml (hunk 1 above)
Comment (RobertGlein) [reply .github/workflows/hls.yml]: The TE HLS script does not work without it.
@aehart Can we do the TE without the download script?

**Comment by RobertGlein** [comment]
> also it looks like the MC test fails at line 500 but the overall test passed?

This is how the MC test was designed.

**Comment by pwittich** [comment]
> > also it looks like the MC test fails at line 500 but the overall test passed?
> 
> This is how the MC test was designed.

hmm, we need to fix this ... do any of the tests fail on invalid matches?

**Pull request merged**